policyengine_us/variables/gov/states/hi/tax/income/hi_state_addback.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class hi_state_addback(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii state income tax addback"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=11"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=31" # State Tax Refund Worksheet
+    )
+    defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        # check eligible?
+        # need go through Form N-11 – State Tax Refund Worksheet
```

Comment:
create a parameter that contains sources of additions in the worksheet

Reply:
Thank you for the review. However, the work on this PR is stopped because of one shortcoming of the current model.